controllers/eventsIndexController.js

```diff
@@ -1,26 +1,20 @@
-const db = require('../db');
+const db = require("../db");
 
 exports.getEventsIndex = async (req, res) => {
-    console.log("GET: Hello getEvents")
-    try { 
-        const collectionEvents = db.getDb().db('softball').collection('events')
-        const allEvents = await collectionEvents.find(
-            {},
-            {_id: 0, date: 1, result: 1}
-        )
-        .sort({ date: -1 })
-        .toArray()
-        console.log('allEvents', allEvents)
-        res
-          .status(200)
-          .json(allEvents);
-    } catch(err) {
-        console.log(err)
-        res
-          .status(500)
-          .json({
-            error: 'db-events-could-not-find',
-            message: err
-        });
-    }
-};
\ No newline at end of file
+  console.log("GET: Hello getEvents");
+  const SortedOrder = (req.query.sort = "desc" ? -1 : 1);
```

Comment:
`SortedOrder` should start with a lower case. It typically means it's a class if it starts with the uppercase.